climada/hazard/storm_europe.py

```diff
@@ -241,7 +241,7 @@ def read_cosmoe_file(self, fp_file, run_datetime, event_date=None,
 
         # read intensity from file
         ncdf = xr.open_dataset(fp_file)
-        ncdf = ncdf.assign_coords(date=('time',ncdf["time"].dt.floor("D")))
+        ncdf = ncdf.assign_coords(date=('time',ncdf["time"].dt.floor("D").values))
```

Comment:
I am not sure to understand the need of this change. Coulb you please comment?

Reply:
Starting with xarray 0.19.0, this will raise an Error. Before, it was only a deprecation warning. https://github.com/pydata/xarray/pull/5630/files#diff-43c76e9be8425b5b6897dcecab4b240c32580447455c0c8c0b9b7fd84ce8a15dL119-R121

This already fails on Jenkins, so I thought we can fix it quickly in this PR.